packages/upload-api/src/usage/record.js

```diff
@@ -0,0 +1,24 @@
+import * as API from '../types.js'
+import * as Provider from '@ucanto/server'
+import { Usage } from '@web3-storage/capabilities'
+
+/** @param {API.UsageServiceContext} context */
+export const provide = (context) =>
+  Provider.provide(Usage.record, (input) => record(input, context))
+
+/**
+ * @param {API.Input<Usage.record>} input
+ * @param {API.UsageServiceContext} context
+ * @returns {Promise<API.Result<API.EgressRecordSuccess, API.EgressRecordFailure>>}
+ */
+const record = async ({ capability }, context) => {
+  const res = await context.usageStorage.record(
```

Comment:
thought: this may be where we take the space DID and look up the customer currently registered as the "provider" - I'm not sure if we have APIs to do this yet so that may need to be added to another interface? 

It's also possible we want to do this resolution inside the `UsageStorage` implementation, in which case we should just pass the space DID through - I'm not sure which makes the most sense but happy to kick it around in higher bandwidth if that's helpful!

Reply:
We will have the customer ID provided during the invocation in Freeway. We could do it here in another moment, but I think that for the first iteration, it should be easy enough to get that in Freeway considering the changes that @Peeja will be working on.

Reply:
I think there's a misunderstanding here - the content commitment will have the space DID, but as far as I know it will not have the account DID - maybe worth a quick checkin to confirm in standup today before digging into this too much deeper, as I could be wrong, but I don't think we include accountDIDs in content commitments (particularly since they contain email addresses and are therefore very sensitive from a GDPR perspective)

Reply:
Correct, no account in location commitment, just space.